pdoc/doc.py

```diff
@@ -282,6 +282,7 @@ def members(self) -> dict[str, Doc]:
                     annotation=doc_f.signature.return_annotation,
                     default_value=empty,
                     taken_from=taken_from,
+                    source=doc_f.source,
```

Comment:
I think this is sensible overall, but as you mention in your comment we should probably also set `source_file` and `source_lines`. _Three_ new attributes in the constructor may be a bit unwieldy, so we could also create the object here and then immediate assign/override those three? Either way - I guess extending the constructor is also reasonable, feel free to make a judgement call here! 😄

Reply:
I removed the additional attribute from the Variable constructor. Two more seemed too much to me. source, source_file and source_lines are added to the freshly created Variable object now.